packages/@contentful--app-scripts/lib/upload/build-upload-settings.js

```diff
@@ -19,13 +19,22 @@ async function buildAppUploadSettings(options) {
       default: '',
     });
   }
+  if (!options.skipActivation) {
```

Comment:
I wouldn't ask for it, only skip if explicitly provided

Reply:
I agree, I wasn't sure about that that's why I added it to be on the safe side. But I will remove it 👍